vbox-2012r2-wmf5.json

```diff
@@ -3,7 +3,6 @@
     {
       "type": "virtualbox-iso",
       "vboxmanage": [
-        [ "modifyvm", "{{.Name}}", "--natpf1", "guest_winrm,tcp,,55985,,5985" ],
```

Comment:
I believe that packer is now dynamically allocating the winrm port.  Before removing that line, every run would be assigned a different port value, so no connection was possible.